021-quarkus-panache-multiple-pus/README.md

```diff
@@ -0,0 +1,2 @@
+# Quarkus - multiple persistence units
+This scenario uses a pair of persistence units, based upon MariaDB and PostgreSQL datasource.
```

Comment:
Please update README with docker commands to run containers. Very useful when trying in dev mode

Reply:
Done.